HotFix.Test/utilities/scheduling/get_active.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using HotFix.Core;
+using HotFix.Utilities;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using static System.DayOfWeek;
+
+namespace HotFix.Test.utilities.scheduling
+{
+    [TestClass]
+    public class get_active
+    {
+        // |  M  |  T  |  W  |  T  |  F  |  S  |  S  |
+        // -->|     |<--->|     |<--------->|     |<--
+        //  ^    ^     ^     ^        ^   ^         ^ 
+        //  A    B     C     D        E   F         G 
+
+        private List<ISchedule> _schedules;
+
+        [TestInitialize]
+        public void Setup()
+        {
+            _schedules = new List<ISchedule>
+            {
+                new Schedule
+                {
+                    Name = "Tuesday",
+                    OpenDay = Tuesday,
+                    OpenTime = TimeSpan.FromHours(10),
```

Comment:
Parsing a timespan would be clearer in these setups: `TimeSpan.Parse("10:00:00")` etc